### ISSUE TITLE
Extract swingset/tendermint stats from testnet and graph over time

### ISSUE THREAD

**Issue by rowgraus** [Agoric/agoric-sdk#2220]
Swingset currently maintains DB keys with a number of statistics, like how many cranks have been executed, how many objects and promises are present in the kernel tables, etc. These roughly correspond to resource usage.

The task is to extend cosmic-swingset to make these counters visible on a local-interface-only HTTP endpoint, in a format suitable for a Prometheus client to fetch them periodically. Once that is in place, we want to build automation on our testnet to fetch and graph this data over time. My main goal is to see the rate-of-change graphed over time, so we can correlate it with intentional optimizations (and accidental de-optimizations) we make in the code.

We would use this in conjunction with OS-level metrics (like the memory used by the cosmic-swingset process, the file size of the swingset database, and the number of CPU seconds consumed by the process), as well as tendermint-level data like the time spent executing transactions for each block.

cc @michaelfig for the Prometheus and cosmic-swingset bits, @FUDCo for the code in swingset that manages these counters

**Comment by rowgraus** [comment]
@warner is this a duplicate of the work you and @michaelfig have already done?  If so, should we close it?

**Comment by dckc** [comment]
> @warner is this a duplicate of the work you and @michaelfig have already done?

I'm reasonably confident that it is.

> If so, should we close it?

Yes. If I have missed anything, feel free to reopen it.
